GNUmakefile

```diff
@@ -87,7 +87,7 @@ dependency/sources/extract-libtermkey: dependency/sources/$(LIBTERMKEY).tar.gz
 	tar xzf $< -C $(dir $<)
 	touch $@
 
-dependency/sources/build-libtermkey: dependency/sources/extract-libtermkey
+dependency/sources/build-libtermkey: dependency/sources/extract-libtermkey dependency/sources/install-libncurses
```

Comment:
This makes sense because libtermkey depends on curses. However the other change seems wrong, why should the lua build depend on curses?